app/Http/Controllers/Auth/AuthController.php

```diff
@@ -45,7 +45,7 @@ protected function create(array $data)
                 'number' => $data['phone_number'],
             ]);
 
-            $this->dispatch(new VerifyPhoneNumberOwnership($number));
+            $this->dispatch(new VerifyPhoneNumber($number));
```

Comment:
I'll keep reading, but I was under the impression each verification had unique content to the text messages they sent out, which is why they originally had different Jobs serving them.